### PR TITLE
feat: Improve innreise status using new fields from backend

### PR DESCRIPTION
**Pull request by SveinungFjaer**

### CODE REVIEW

File: components/home/lists/lists.html [hunk 1]
```diff
@@ -52,30 +52,46 @@
...
-                    <div>
-                        <div ng-if="innreiseSistOppdatert">Innreisedata sist oppdatert {{innreiseSistOppdatert}}.</div>
-                        <span ng-if="!innreiseSistOppdatert">Innreisedata har ikke blitt hentet ennå. </span>
-                        <div ng-if="innreiseProvesvarSistOppdatert"> Prøvesvardata sist hentet {{innreiseProvesvarSistOppdatert}}.</div>
-                        <span ng-if="showNoProvesvardataHentet()">Prøvesvar har ikke blitt hentet ennå. </span>
-                        <span ng-if="!harTilgangTilProvesvar">Du har ikke tilgang til å synkronisere prøvesvar. </span>
-                        <span ng-if="hasStartedSync"> Henting av prøvesvar ble nylig startet, og kan ikke startes på nytt akkurat nå.</span>
+                    <div ng-if="!innreiseStatusQueryFailed">
+                        <div ng-if="innreiseStatus === 'AVSLUTTET'">Innreisedata sist oppdatert {{innreiseSistOppdatert}}.</div>
+                        <div ng-if="innreiseStatus === 'KJORER'">Ny innreisedata lastes inn. Dette kan ta flere minutter. Oppdater siden for å hente inn ny data.</div>
```
Comment (helgeskjellevik) [inline components/home/lists/lists.html:65]: Nye innreisedata lastes inn

File: components/home/lists/lists.html [hunk 2]
```diff
@@ -52,30 +52,46 @@
...
+                        <div ng-if="innreiseStatus === 'FEIL'" class="error">Det skjedde en feil ved sist innlesning av innreisedata.</div>
```
Comment (helgeskjellevik) [inline components/home/lists/lists.html:66]: feil ved siste innlesing*

 (eller er det innlesning)

File: components/home/lists/lists.html [hunk 3]
```diff
@@ -52,30 +52,46 @@
...
                     </div>
-                    <button ng-if="showProvesvarSyncButton()" ng-click="syncLabTests()" class="btn btn-primary" style="margin-left: 1em">Hent prøvesvar nå</button>
-                </div>
-                <span ng-if="labTestNotActivated">
                     <hr>
-                    Prøvesvar er ikke aktivert for din kommune.
+                    <div style="display: flex" ng-if="!innreiseStatusQueryFailed">
+                        <div style="align-self: center">
+                            <div ng-if="provesvarStatus === 'AVSLUTTET'">Prøvesvardata sist hentet {{innreiseProvesvarSistOppdatert}}.</div>
+                            <div ng-if="provesvarStatus === 'KJORER'">Ny prøvesvardata lastes inn. <br />Dette kan ta flere minutter. Oppdater siden for å hente inn ny data.</div>
```
Comment (helgeskjellevik) [inline components/home/lists/lists.html:74]: Nye prøvesvardata

File: components/home/lists/lists-controller.js [hunk 4]
```diff
@@ -429,51 +429,66 @@ trackerCapture.controller('ListsController',function(
...
                 try{
                     userId = JSON.parse(sessionStorage.USER_PROFILE).id
                 }
                 finally {}
-                $scope.hasStartedSync = true;
+                $scope.proveSvarSyncIsLoading = true;
                 FNrLookupService.startLabTestSync($scope.selectedOrgUnit.code, userId).then(function(svar){
-                    if(!svar) {
-                        $scope.hasStartedSync = false;
+                    $scope.proveSvarSyncIsLoading = false;
```
Comment (Bekkalizer) [inline components/home/lists/lists-controller.js:448]: Indentation :S

Comment (SveinungFjaer) [reply components/home/lists/lists-controller.js:448]: Is correct, isn't it?

Comment (SveinungFjaer) [reply components/home/lists/lists-controller.js:448]: or, the whole block maybe...

Comment (SveinungFjaer) [reply components/home/lists/lists-controller.js:448]: No, seems correct to me. I'll remove an extra linebreak...

Comment (Bekkalizer) [reply components/home/lists/lists-controller.js:447]: You are right :) Got tricked by the lines above

**Review by Bekkalizer — APPROVED** [review]

**Pull request merged**